tools/testing/selftests/bus1/test-io.c

```diff
@@ -370,7 +370,8 @@ int test_io(void)
 	fprintf(stderr, "it took %lu ns + %lu ns for one destination\n", base,
 		test_iterate(100000, 1, 1024) - base);
 
-	for (unsigned int i = 1; i < 9; ++i) {
+	unsigned int i;
```

Comment:
Yeah, this is not the kernel style. My bad. Move it all the way to the top of the block though.